include/http/request_handler.js

```diff
@@ -356,6 +356,17 @@ module.exports = function RequestHandlerModule(pb) {
                 }
 
                 this.resp.setHeader('content-type', contentType);
+                let enabledSecureCookie = this.req && this.req.siteObj && this.req.siteObj.enabledSecureCookie;
+                enabledSecureCookie = (enabledSecureCookie == undefined) ? true : enabledSecureCookie;
```

Comment:
minor:
the ```enabledSecureCookie == undefined``` looks not good.
Shenghua proposed that, you can change the variable from ```enabledSecureCookie ``` to ```disableSecureCookie```, like
```let disableSecureCookie = this.req && this.req.siteObj && this.req.siteObj.enabledSecureCookie;```
and ```disableSecureCookie``` will be false/undefined, then you can remove the next line, and directly check 
```if (!disableSecureCookie && cookied)```